tern/command_lib/snippets.yml

```diff
@@ -54,4 +54,7 @@ dnf:
     - 'check-update'
     - 'clean'
   packages: 'rpm'
-
+rpm:
+   install: '-i'
+   remove: '-e'
+   packages: 'rpm'
```

Comment:
Is this change part of this change set? It seems to me this adds some extra package manager enabling.

Reply:
It is tangentially related to how rpm specifies the package delimiter using `-`.